src/Grammars/MySqlGrammar.php

```diff
@@ -11,6 +11,8 @@ class MySqlGrammar extends BaseMySqlGrammar
      */
     public function wrapJsonSelector($value)
     {
-        return parent::wrapJsonSelector($value);
```

Comment:
Create a request with two levels of attributes, as in the example below.

```
{
    "branch": {
        "name": "Los Angeles"
    }
}
```

Then run the following PMQL:
```
data.branch.name = "Los Angeles"
```
The result was a syntax error.

```
The produced SQL:
data->>"$.branch->name" = Los Angeles

Before this branch
data->>"$.branch.name" = Los Angeles
```